keras/layers/normalization/batch_normalization.py

```diff
@@ -892,9 +899,6 @@ def _compose_transforms(scale, offset, then_scale, then_offset):
                 offset += then_offset
             return (scale, offset)
 
-        # Determine a boolean value for `training`: could be True, False, or
-        # None.
-        training_value = control_flow_util.constant_value(training)
```

Comment:
Thanks for the review! I did try using `training` by itself initially, but it failed due to being a `tf.Tensor`, so I moved `training_value` up from this line. Presumably this is a workaround for some behavior in `_get_training_value()`, but that should probably be fixed in another PR by someone more familiar with this codebase than me.